epfast-annotations/src/main/java/com/exactpro/epfast/CreatorService.java

```diff
@@ -0,0 +1,15 @@
+package com.exactpro.epfast;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.ServiceLoader;
+
+public class CreatorService {
+    public static List<ICreator> providers() {
```

Comment:
I'd rather
```java
    public static Collection<ICreator> providers() {
        return streamIterable(ServiceLoader.load(ICreator.class)).collect(Collectors.toList());
    }
    
    private static <T> Stream<T> streamIterable(Iterable<T> iterable) {
        return StreamSupport.stream(iterable.spliterator(), false);
    }
```

Reply:
streamIterable can be placed in some utility class com.exactpro.ustil.Stream for example